cli/tests/testdata/test/shuffle.out

```diff
@@ -1,6 +1,6 @@
-Check [WILDCARD]/test/shuffle/foo_test.ts
-Check [WILDCARD]/test/shuffle/baz_test.ts
 Check [WILDCARD]/test/shuffle/bar_test.ts
+Check [WILDCARD]/test/shuffle/baz_test.ts
```

Comment:
We can maintain the previous behaviour of re-shuffling checks, just did not see no value in it.

Reply:
That sounds good to me